yesod-websockets/ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.4.2
```

Comment:
This requires a minor version bump due to expansion of the API (in order words, this should be 0.2.5)